offix-offline/src/main/java/org/aerogear/offixoffline/Extension.kt

```diff
@@ -0,0 +1,51 @@
+package org.aerogear.offixoffline
+
+import android.util.Log
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.ApolloClient
+import com.apollographql.apollo.api.Mutation
+import com.apollographql.apollo.api.Operation
+
+/* Extension function on ApolloClient which will be used by the user while making a call request.
+   @receiver parameter is ApolloClient on which the call will be made by the user.
+   @param mutation which will be stored in the list if network connection is not there.
+   @param callback which will be stored in the list if network connection is not there.
+ */
+fun ApolloClient.enterQueue(
```

Comment:
Let's call this method `enqueue()`; sounds much better than `enterQueue` or `offQueue`